sdk/state/payment.go

```diff
@@ -158,14 +162,18 @@ var ErrUnderfunded = fmt.Errorf("account is underfunded to make payment")
 // on the state of the close agreement signatures.
 func (c *Channel) validatePayment(ca CloseAgreement) (err error) {
 	// If the channel is not open yet, error.
-	if c.latestAuthorizedCloseAgreement.isEmpty() {
+	cs, err := c.State()
+	if err != nil {
+		return fmt.Errorf("getting channel state: %w", err)
+	}
+	if cs < StateOpen {
```

Comment:
👍🏻 I think using the state here is good, however I think the condition needs to be `cs != StateOpen`, because any non-open state should not be supported here.